Gruntfile.js

```diff
@@ -10,24 +10,26 @@ module.exports = function(grunt) {
     concat: {
       dist: {
         options: {
-          banner: "(function(){\n",
-          footer: "}());\n"
+          banner: '(function(exports, crypto, Promise, Error, Uint8Array, undefined){\n"use strict";\n\n// supporting Safari and its vendor prefix\nif(!crypto.subtle) crypto.subtle = crypto.webkitSubtle;\n',
```

Comment:
You responded somewhere about `undefined` (your response in somewhere in sea of comments), it's neat trick (in a weird way) but I prefer having all undefined checks being done using `typeof(foo) == "undefined"` (which seems like the right thing to do). Never relying on whether `undefined` is defined or not seems cleaner.